cmd/sandbox-ce/compose/compose.go

```diff
@@ -135,5 +135,6 @@ func Run(ctx context.Context, arg ...string) error {
 		return err
 	}
 
-	return comp.Run(ctx, arg...)
+	newArgs := append([]string{"--project-name", "srcd"}, arg...)
```

Comment:
github doesn't show conflict but current master already set project name

Reply:
in master it was a different method, so there was no conflict but the result would be the same arg added twice. I have removed it here, rebased on top of master and squashed.